sktime/forecasting/adapters/_autots.py

```diff
@@ -0,0 +1,449 @@
+
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter settings for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return `"default"` set.
+            There are currently no reserved values for forecasters.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+        params = {
+            "model_list": None,
+            "frequency": "infer",
+            "prediction_interval": 0.9,
+            "max_generations": 1,
+            "no_negatives": False,
+            "constraint": None,
+            "ensemble": "auto",
+            "initial_template": "General+Random",
+            "random_seed": 2022,
+            "holiday_country": "US",
+            "subset": None,
+            "aggfunc": "first",
+            "na_tolerance": 1,
+            "metric_weighting": {
+                "smape_weighting": 5,
+                "mae_weighting": 2,
+                "rmse_weighting": 2,
+                "made_weighting": 0.5,
+                "mage_weighting": 0,
+                "mle_weighting": 0,
+                "imle_weighting": 0,
+                "spl_weighting": 3,
+                "containment_weighting": 0,
+                "contour_weighting": 1,
+                "runtime_weighting": 0.05,
+                "oda_weighting": 0.001,
+            },
+            "drop_most_recent": 0,
+            "drop_data_older_than_periods": 100000,
+            "transformer_list": "auto",
+            "transformer_max_depth": 6,
+            "models_mode": "random",
+            "num_validations": "auto",
+            "models_to_validate": 1,
+            "max_per_model_class": None,
+            "validation_method": "seasonal 1",
+            "min_allowed_train_percent": 0.5,
+            "remove_leading_zeroes": False,
+            "prefill_na": None,
+            "introduce_na": None,
+            "preclean": None,
+            "model_interrupt": True,
+            "generation_timeout": None,
+            "current_model_file": None,
+            "verbose": -1,
+            "n_jobs": -2,
+            "model_name": "GLS",
+        }
+        return params
+
+    def _instantiate_model(self):
+        """Instantiate the autoTS model.
+
+        Returns
+        -------
+            self: This function returns a reference to the instantiated model object.
+        """
+        if type(self.ensemble) is not list:
```

Comment:
I think `isinstance` is recommended more popularly.

Reply:
yes - though there is an important difference which does not matter in this case, just pointing it out - `type` equals checks instance type equality, whereas `isinstance` is also `True` if it is an instance of a child class.